tests/test_packages/test_skills_integration/test_coin_price.py

```diff
@@ -53,14 +53,19 @@ class TestCoinPriceSkill(AEATestCaseEmpty):
 
     def test_coin_price(self):
         """Run the coin price skill sequence."""
+
+        coin_price_feed_aea_name = "coin_price_feed_aea"
+        self.create_agents(coin_price_feed_aea_name)
```

Comment:
Since you're using `AEATestCaseEmpty` this is not needed. You can retrieve agent name from `coin_price_feed_aea_name = self.agent_name`